spec/rollout_spec.rb

```diff
@@ -15,9 +15,29 @@
   context "a partner organisation user" do
     let(:user) { build(:partner_organisation_user) }
 
-    it "is part of the partner_organisation_users group and not the beis_users group" do
+    it "is part of the partner_organisation_users group and not the beis_users group", :use_original_rollout do
       expect(ROLLOUT.active_in_group?(:beis_users, user)).to be false
       expect(ROLLOUT.active_in_group?(:partner_organisation_users, user)).to be true
     end
   end
+
+  describe "#ispf_in_stealth_mode_for_group?" do
+    it "provides a more readable interface to check if the feature is enabled for that group" do
+      mock_feature = double(:feature, groups: [:real_group])
+      allow(ROLLOUT).to receive(:get).with(:ispf_fund_in_stealth_mode).and_return(mock_feature)
+
+      expect(ispf_in_stealth_mode_for_group?(:real_group)).to be true
+      expect(ispf_in_stealth_mode_for_group?(:fake_group)).to be false
+    end
+  end
+
+  describe "#ispf_in_stealth_mode_for_user?" do
+    let(:user) { double(:user) }
+
+    it "provides a more readable interface to check if the feature is enabled for that user" do
+      expect(ROLLOUT).to receive(:active?).with(:ispf_fund_in_stealth_mode, user)
```

Comment:
Any reason you went for the `expect(x).to receive` rather than calling the method followed by `expect(x).to have_received`? More for my own curiosity than this not being the correct approach.

Reply:
No particular reason, probably contagion from the spec above, and lack of a strong preference for `have_received`.

I used to be a proponent of `have_received` for reasons of following the arrange-act-assert order, but I think `have_received` didn't work for message chains so I started using it less. Not the case here, but the style stuck.